tests/test_authorizer.py

```diff
@@ -361,26 +418,16 @@ def test_refresh_with__valid_otp(self):
         self.assertEqual(set(["*"]), authorizer.scopes)
         self.assertTrue(authorizer.is_valid())
 
-    def test_refresh_with__invalid_otp(self):
+    def test_refresh_with_scopes(self):
```

Comment:
Can you name this `test_refresh__with_scopes` please and insert it in the correct sorted order?